assets/javascripts/discourse/templates/modal/login-prompt.hbs

```diff
@@ -0,0 +1,31 @@
+{{#d-modal-body title="expose_emails_in.login_prompt.title" class="login-prompt"}}
+  <div>
+    <h3>
+      {{#if model.stranger_danger}}
+        {{i18n "expose_emails_in.login_prompt.body_stranger"}}
+      {{else}}
+        {{i18n "expose_emails_in.login_prompt.body"}}
+      {{/if}}
+      {{d-button  class="btn-primary login-button"
```

Comment:
I don't know if this is some sort of standard code style for handlebars, but you seem to have two spaces here.